server/models/productModel.js

```diff
@@ -0,0 +1,32 @@
+const { DataTypes: dt } = require("sequelize");
+const db = require("./index");
+
+const ProductModel = db.define("Product", {
+	id: {
+		type: dt.INTEGER,
+		primaryKey: true,
+		autoIncrement: true,
+	},
+	current_price: {
+		type: dt.DECIMAL(10, 2),
+		allowNull: false,
+	},
+	previous_price: {
+		type: dt.DECIMAL(10, 2),
+	},
+	link: {
+		type: dt.STRING,
+		allowNull: false,
+		unique: true,
+	},
+	company: {
+		type: dt.STRING,
+		allowNull: false,
+	},
```

Comment:
What does company to represent here?

Reply:
Where the item is being sold through (e.g. amazon, craigslist). It's probably something we could derive from the link, but I thought it migjt be good to have it stored as its own field as well.